src/selection/selectChildren.js

```diff
@@ -0,0 +1,19 @@
+import matcher from "../matcher.js";
+
+var filter = Array.prototype.filter;
+
+function children() {
+  return this.children;
+}
+
+function childrenFilter(match) {
+  match = typeof match === "function" ? match : matcher(match);
+  return function() {
+    return filter.call(this.children, function(e) { return match.call(e); });
```

Comment:
Same comment as above. Since the *match* function will be called on every child of every selected element (rather than every selected element), it doesn’t need to conform to the standard D3 convention, and instead should follow the array convention.

```suggestion
    return filter.call(this.children, match);
```